<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LicenseDialog</class>
 <widget class="QDialog" name="license_dialog_">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>License Agreement</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Resources/assets/icon_48x48.png</normaloff>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="vertical_layout_">
   <item>
    <widget class="QTextEdit" name="text_edit_">
     <property name="html">
      <string>
       &lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
       &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
       p, li { white-space: pre-wrap; }
       &lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
       &lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;/docs/License.htm&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
      </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_" native="true">
     <layout class="QHBoxLayout" name="horizontal_layout_">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontal_spacer_">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="decline_button_">
        <property name="text">
         <string>Decline</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="agree_button_">
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>agree_button_</tabstop>
  <tabstop>decline_button_</tabstop>
  <tabstop>text_edit_</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>

